pennylane/ops/qubit/observables.py

```diff
@@ -349,54 +347,126 @@ class Projector(Observable):
     * Gradient recipe: None
 
     Args:
-        basis_state (tensor-like): binary input of shape ``(n, )``
-        wires (Iterable): wires that the projector acts on
+        state (tensor-like): input state
+        wires (Iterable): wires that the projector acts on.
         do_queue (bool): Indicates whether the operator should be
             immediately pushed into the Operator queue (optional).
             This argument is deprecated, instead of setting it to ``False``
             use :meth:`~.queuing.QueuingManager.stop_recording`.
-        id (str or None): String representing the operation (optional)
+        id (str or None): String representing the operation (optional).
+
+    **Example**
+
+    In the following example we consider projectors over two states: the :math:`|00\rangle` and the
+    :math:`|++\rangle`. Since the first one is in the computational basis, we create its projector
+    directly from its basis state representation, which is, ``three_state=[1, 1]``. For the latter,
```

Comment:
```suggestion
    In the following example we consider projectors over two states: :math:`|00\rangle` and
    :math:`|++\rangle`. Since the first one is in the computational basis, we create its projector
    directly from its basis state representation, which is, ``zero_state=[0, 0]``. For the latter,
```